docs/cli.md

```diff
@@ -0,0 +1,38 @@
+# Promscale command line doc
+
+This document gives you information about the configuration flags supported by promscale.
+You can also find information on flags with `promscale_<version> -help` 
+
+## General flags
+| Flag | Type | Default | Description |
+|------|:-----:|:-------:|:-----------:|
+| install-timescaledb | boolean | true | Install or update TimescaleDB extension. |
+| leader-election-pg-advisory-lock-id | integer | 0 | Leader-election based high-availability. It is based on PostgreSQL advisory lock and requires an unique advisory lock ID per high-availability group. |
+| leader-election-pg-advisory-lock-prometheus-timeout | slack/duration | -1 | Prometheus timeout duration for leader-election high-availability. The connector will resign if the associated Prometheus instance does not respond within the given timeout. Note: Make sure that only one Prometheus instance talks to the adapter. Timeout value should be co-related with Prometheus scrape interval but enough to prevent random flips. |
+| log-format | string | logfmt | Log format to use from [ "logfmt", "json" ]. |
+| log-level | string | debug | Log level to use [ "error", "warn", "info", "debug" ]. |
+| migrate | string | true | Update the Prometheus SQL to the latest version. Valid options are: [true, false, only]. |
+| read-only | boolean | false | Read-only mode for the connector. Operations related to writing or updating the database are restricted. It is useful when pointing adapter to the read replica. |
+| scheduled-election-interval | duration | 5 seconds | Interval at which scheduled election runs. This is used to select a leader and confirm that we still holding the advisory lock. |
+| use-schema-version-lease | boolean | true | Used to prevent race conditions during migration. |
+| web-cors-origin | string | .* |  Regex for CORS origin. It is fully anchored. Example: 'https?://(domain1|domain2)\.com' |
+| web-listen-address | string | :9201 | Address to listen on for web endpoints. |
+| web-telemetry-path | string | `/metrics` | Web endpoint for exposing Promscale's Prometheus metrics. |
+
+## Database flags
+
+| Flag | Type | Default | Description |
+|------|:-----:|:-------:|:-----------:|
+| async-acks | boolean | false | Acknowledge asynchronous inserts. If this is true, the inserter will not wait after insertion of metric data in the database.|
```

Comment:
```suggestion
| async-acks | boolean | false | Acknowledge asynchronous inserts. If this is true, the inserter will not wait after insertion of metric data in the database. This increases throughput at the cost of a small chance of data loss. | 
```

Reply:
How is there gonna be a data loss? Do we discard samples after a time window?

Reply:
If the insert is issued, the ack, is returned, and the DB crashes before the insert is written to disk. That's why it's off by default.